setup.py

```diff
@@ -23,5 +23,5 @@
         'Programming Language :: Python :: 3.4',
     ],
     test_suite='runtests.runtests',
-    tests_require=['responses>=0.5']
+    tests_require=['responses>=0.5', 'mock>=1.3']
```

Comment:
if you're going to try to use the built in unittest.mock in 3.4, you would probably only want to require mock in <3.4